charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -80,6 +80,31 @@ Build config file for deployment OpenTelemetry Collector
 {{- tpl (toYaml $config) . }}
 {{- end }}
 
+{{/*
+Build config file for daemonset OpenTelemetry Collector
+*/}}
+{{- define "opentelemetry-collector.statefulsetConfig" -}}
```

Comment:
Do we need this?  The `mode=statefulset` configuration uses the deployment config since deployments and statefulsets don't need different collector configuration.

Reply:
Resolved:
https://github.com/nhuray/opentelemetry-helm-charts/commit/28a2ad2d4c46f9808a2102b80a45f84cd92f3c3d